src/composables/multisigAccounts.ts

```diff
@@ -149,36 +164,36 @@ export function useMultisigAccounts({ store }: UseMultisigAccountsOptions) {
 
     const result = (await Promise.all(
       rawMultisigData.map(async ({ contractId, gaAccountId, ...otherMultisig }) => {
-        const contractInstance = await sdk.getContractInstance({
-          aci: SimpleGAMultiSigAci,
-          contractAddress: contractId,
-        });
-
-        const nonce = (await contractInstance.methods.get_nonce()).decodedResult;
-        const signers = (await contractInstance.methods.get_signers()).decodedResult;
-        const version = (await contractInstance.methods.get_version()).decodedResult;
-
-        const consensus = (
-          await contractInstance.methods.get_consensus_info()
-        ).decodedResult as any;
-
-        if (consensus.tx_hash) {
-          consensus.tx_hash = Buffer.from(consensus.tx_hash).toString('hex');
+        try {
+          const contractInstance = await getContractInstance(contractId, sdk);
+          const nonce = await getNonce(contractInstance);
+          const signers = await getSigners(contractInstance);
+          const version = await getVersion(contractInstance);
+          const consensus = await getConsensus(contractInstance);
+
+          if (consensus.tx_hash) {
+            consensus.tx_hash = Buffer.from(consensus.tx_hash).toString('hex');
+          }
+          consensus.expiration_height = Number(consensus.expiration_height);
+          consensus.confirmations_required = Number(consensus.confirmations_required);
+
+          return {
+            ...camelcaseKeysDeep(consensus),
+            ...otherMultisig,
+            nonce: Number(nonce),
+            signers,
+            version,
+            contractId,
+            balance: (gaAccountId && (await sdk.balance(gaAccountId))) || 0,
+            address: gaAccountId,
+            multisigAccountId: gaAccountId,
+          };
+        } catch (e) {
+          /* Node might throw nonce mismatch error, skip the current account update
+          * return the existing data and account details will be updated in the next poll.
+          */
```

Comment:
Please use correct DocBlock.
```
/**
 *
 */
```
This VS Code plugin is really helpful in this field: https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments